prosper/crons/FetchMapInfo.py

```diff
@@ -0,0 +1,100 @@
+'''FetchMapInfo.py: a script for saving the map statistics from EVE Online XML API'''
+
+from os import path #TODO: path->plumbum?
+
+import requests
+import pandas
+import xml.etree.ElementTree as ET
+from plumbum import cli
+
+from prosper.common.prosper_logging import create_logger
+from prosper.common.prosper_config import get_config
+
+HERE = path.abspath(path.dirname(__file__))
+ME = __file__.replace('.py', '')
+CONFIG_ABSPATH = path.join(HERE, 'cron_config.cfg')
+
+config = get_config(CONFIG_ABSPATH)
+logger = create_logger(
+    'debug-FetchMapInfo',
+    HERE,
+    config
+)
+
+## MAP INFO GLOBALS ##
+XML_BASE_URL = config.get(ME, 'XML_endpoint')
+XML_FILES = config.get(ME, 'files')
+
+
+def get_map_info_from_eve(
+    file
+):
+    url = XML_BASE_URL + file
+    try:
+        request = requests.get(
+            url
+        )
```

Comment:
add `request.raise_for_status()` and we can remove `if request.status_code == requests.codes.ok:`